src/workos.ts

```diff
@@ -141,13 +151,20 @@ export class WorkOS {
     path: string,
     options: GetOptions = {},
   ): Promise<{ data: Result }> {
+    const requestHeaders: any = {};
```

Comment:
I know this was just copied from the other helpers, so don't feel the need to change this, but I'd love if we replaced these `any` `requestHeaders` with `Record<string, string>` instead:

```suggestion
    const requestHeaders: Record<string, string> = {};
```